app/src/main/java/com/github/polypoly/app/base/game/rules_and_lobby/GameRules.kt

```diff
@@ -0,0 +1,37 @@
+package com.github.polypoly.app.base.game.rules_and_lobby
+
+import com.github.polypoly.app.base.game.location.Zone
+import com.github.polypoly.app.map.LocationRepository
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.hours
+
+/**
+ * A class that represent the rules of a [Game]
+ * @property gameMode The game mode of the future game
+ * @property minimumNumberOfPlayers The minimum number of players needed to start the game
+ * @property maximumNumberOfPlayers The maximum number of players that can join the lobby
+ * @property roundDuration The duration of a round in the
+ * @property maxRound The maximum number of round before the game end. This settings is available
+ * only in RICHEST_PLAYER mode. Is null if an other game mode is selected.
+ * @property gameMap The map of the game whit the different zones available
+ * @property initialPlayerBalance The initial balance of money of the players
+ */
+data class GameRules (
+    val gameMode: GameMode = GameMode.RICHEST_PLAYER,
+    val minimumNumberOfPlayers: Int = 3,
+    val maximumNumberOfPlayers: Int = 7,
+    val roundDuration: Duration = 2.hours,
```

Comment:
As we discussed together, `roundDuration` should better be an integer than a `Duration`. Let's keep it that way because i already changed it in a PR

Reply:
done